core/event.c

```diff
@@ -1415,6 +1429,12 @@ struct uwsgi_timer *event_queue_ack_timer(int id) {
 }
 #endif
 
+#if !HAS_ADD_FD_ACCEPT
```

Comment:
`#if` can be used for undefined.  But I'll change to use ifdef if you prefer it.